src/core/free_for_all_game_mode_system.cpp

```diff
@@ -61,7 +61,7 @@ void FreeForAllGameModeSystem::OnStartGameMode( core::StartGameModeEvent const&
     ::engine::Engine::Get().SetEnabled< ::core::CaptureTheFlagGameModeSystem>( false );
     ::engine::Engine::Get().SetEnabled< ::core::RogueGameModeSystem>( false );
 
-    mScene.Load( RootModel::Get()["level"]["getselected"].operator std::string() );
+    mScene.Load( RootModel::Get()["level"].operator std::string() );
```

Comment:
Please use getters and setters from the corresponding system instead of the model. (everywhere - won't mark the other occurrances)